easybuild/easyconfigs/p/PCRE/PCRE-8.39-GCCcore-5.4.0.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'PCRE'
+version = '8.39'
+
+homepage = 'http://www.pcre.org/'
+description = """The PCRE library is a set of functions that implement regular expression pattern matching using
+ the same syntax and semantics as Perl 5."""
+
+toolchain = {'name': 'GCCcore', 'version': '5.4.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['ccdf7e788769838f8285b3ee672ed573358202305ee361cfec7a4a4fb005bbc7']
+
+builddependencies = [
+    # use same binutils version that was used when building GCCcore toolchain
+    ('binutils', '2.27', '', True),
```

Comment:
@vanzod No need to hardcode to `dummy` toolchain here, and it's OK to make this single-line nowadays imho...

```python
builddependencies = [('binutils', '2.27')]